packages/dd-trace/src/plugins/log_plugin.js

```diff
@@ -17,6 +24,11 @@ function messageProxy (message, holder) {
         return holder.dd
       }
 
+      // Special handling for Error stack property in Node.js 22
+      if (p === 'stack' && originalStack !== undefined) {
+        return originalStack
+      }
```

Comment:
```suggestion
      // This is a workaround for a V8 bug that surfaced in Node.js 22
      if (p === 'stack') {
        return target.stack
      }
```

This seems to be the only change we should do here. The issue seems to be in V8 itself that the stack is not calculated when being called with Reflect.get inside of a proxy.

Reply:
Awesome!

Reply:
Note: this is not completely safe to do but I think it's a good short term solution.

A safer way would probably be to check if Reflect.get returns undefined and if that's the case, check if the property is `'stack'` and return that. That way the access would use Reflect.get for all cases where we actually do get a value.